### PR TITLE
[ MOSIP-24328 ] Updated Localhost mapping script and info in k8s-infra.

### PR DESCRIPTION
**Pull request by Mohanraj209**

### CODE REVIEW

**Comment by syedsalman3753** [comment]
@Mohanraj209 please provide Ticket ID in the PR

**Comment by syedsalman3753** [comment]
Get the documentation review by @SwatiPeriwal

File: mosip/on-prem/README.md
Comment (SwatiPeriwal) [inline mosip/on-prem/README.md:47]: ## Mapping localhost

Comment (SwatiPeriwal) [inline mosip/on-prem/README.md:47]: Below is some information about localhost mapping in brief:

File: mosip/on-prem/README.md
Comment (SwatiPeriwal) [inline mosip/on-prem/README.md:49]: An issue with the nodal instances was observed as localhost mapping was not present in the hosts file of the system.

File: mosip/on-prem/README.md
Comment (SwatiPeriwal) [inline mosip/on-prem/README.md:52]: Note: An ansible script was written and the task was automated to reduce the effort of checking all the nodes of the cluster. This was extremely tedious and time consuming manually.

File: mosip/on-prem/README.md
Comment (SwatiPeriwal) [inline mosip/on-prem/README.md:54]: **To check localhost mapping is present in all cluster nodes**

File: mosip/on-prem/README.md
Comment (SwatiPeriwal) [inline mosip/on-prem/README.md:58]: **To view the ansible script or make any changes**

File: mosip/on-prem/README.md
Comment (SwatiPeriwal) [inline mosip/on-prem/README.md:50]: Issue: Localhost mapping i.e. ```127.0.0.1    localhost``` might not being present within ```/etc/hosts``` file of the system.

File: mosip/on-prem/README.md
Comment (SwatiPeriwal) [reply mosip/on-prem/README.md:52]: Resolution: An ansible script was written and the task was automated to reduce the effort of checking all the nodes of the cluster. This was extremely tedious and time consuming manually.

**Review by SwatiPeriwal — CHANGES_REQUESTED** [review]
Added comments.. please verify and update changes accordingly.

**Comment by Mohanraj209** [comment]
@SwatiPeriwal mam i have made changes pls check once and approve it.

**Review by SwatiPeriwal — APPROVED** [review]
Approved changes..

**Review by syedsalman3753 — APPROVED** [review]

**Review by ckm007 — COMMENTED** [review]
# Mapping localhost changes defination should be added in troubleshooting docs.

**Review by ckm007 — APPROVED** [review]

**Pull request merged**
